bin/riemann-elasticsearch

```diff
@@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
   end
 
   def health_url
-    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
+    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
```

Comment:
Yeah, feel free to remove one!
On May 29, 2015 9:19 PM, "Kyle Kingsbury" <notifications@github.com> wrote:

> In bin/riemann-elasticsearch
> <https://github.com/aphyr/riemann-tools/pull/115#discussion_r31356169>:
>
> > @@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
> >    end
> >
> >    def health_url
> > -    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
> > +    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
>
> Ah, that too--I actually meant that by default, this is gonna hit
> //_cluster/health, not /_cluster/health.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aphyr/riemann-tools/pull/115/files#r31356169>.
>